### ISSUE TITLE
Faker issue

### ISSUE THREAD

**Issue by afedosenko** [pyupio/safety#297]
* safety version: 1.8.7
* Python version: 3.5.9
* Operating System: macOS, Ubuntu

### Description

The `safety check` command show next:
<img width="577" alt="Снимок экрана 2020-04-10 в 19 12 40" src="https://user-images.githubusercontent.com/1413044/79005331-46e8e880-7b5f-11ea-9fcd-abaab2b2a215.png">
But  the "faker" package has only the 2.0.5 and less versions from 2.x versions, then 3.x versions.
https://github.com/joke2k/faker/releases

**Comment by afedosenko** [comment]
Json:
```
[
    [
        "faker",
        "<2.1.2",
        "2.0.5",
        "Faker 2.1.2: `bundler-audit` has identified that i18 has a security vulnerability, that has been fixed in the 0.8 version.",
        "37658"
    ]
]
```

**Comment by harlekeyn** [comment]
You're right. Safety confused [Faker (for Python)](https://github.com/joke2k/faker) with [Faker (for Ruby)](https://github.com/faker-ruby/faker).

Fixed.
